lms/views/api/sync.py

```diff
@@ -0,0 +1,35 @@
+import hashlib
+
+from pyramid.view import view_config
+
+from lms.models import HGroup
+
+
+@view_config(
+    route_name="api.sync", request_method="POST", renderer="json", permission="sync_api"
+)
+def sync(request):
+    canvas_api_svc = request.find_service(name="canvas_api_client")
+    lti_h_svc = request.find_service(name="lti_h")
+
+    context_id = request.json["course"]["context_id"]
+    custom_canvas_course_id = request.json["course"]["custom_canvas_course_id"]
+    tool_consumer_instance_guid = request.json["lms"]["tool_consumer_instance_guid"]
```

Comment:
We should probably add a validation schema to the view for good measure, rather than just assuming all the params are present and correct, but it's not like the frontend is ever going to be calling it invalidly anyway

Reply:
There are arguably missing domain model objects here: `LMS` and `LTICourse`. A validation schema could even create them for the view so that `request.parsed_params["course"]` would be a `models.LTICourse` object and so on. But I'm not sure it's really worthwhile introducing them at this point.